exec/java-exec/src/main/resources/rest/static/js/querySubmission.js

```diff
@@ -11,6 +11,38 @@
  *  language governing permissions and limitations under the License.
  */
 
+//Elements for Timer in LoadingModal
+var elapsedTime = 0;
+var delay = 1000; //msec
+var timeTracker = null; //Handle for stopping watch
+
+//Padding for time
+function str_pad_left(string,pad,length) {
+    return (new Array(length+1).join(pad)+string).slice(-length);
```

Comment:
How about below:

```
function str_pad_left(timeVal, length) {
  return timeVal.toString().slice(-length);
}
```

Reply:
That actually shows only one digit for values less than 10. Modifying it to 
`let prettyTime = ("0" + minutes).slice(-2)+':'+ ("0" + seconds).slice(-2);`